ethereum_history_api/circuits/lib/src/fixtures/sepolia/dencun/some_blobGasUsed/header.nr

```diff
@@ -14,8 +14,8 @@ global receipts_root = [
     20,51,250,45,179,111,83,74,189,191,127,230,184,133,54,169,122,191,237,126,94,154,208,238,136,168,105,95,151,127,18,87
 ];
 
-global encoded_length: Field = 549;
-global encoded_data = [249,2,34,160,165,182,222,31,41,42,44,192,198,155,203,225,71,18,212,133,11,53,58,106,47,231,97,242,55,238,16,220,103,206,197,121,160,29,204,77,232,222,199,93,122,171,133,181,103,182,204,212,26,211,18,69,27,148,138,116,19,240,161,66,253,64,212,147,71,148,234,126,66,199,201,38,46,104,49,191,167,190,113,214,122,165,82,62,39,189,160,135,0,211,62,157,200,170,209,112,246,212,67,165,196,92,229,106,89,216,165,102,187,73,198,153,11,205,255,174,199,237,40,160,66,16,117,172,101,103,65,103,42,223,41,72,96,187,242,164,229,191,223,43,152,144,145,62,53,125,125,229,148,153,188,18,160,20,51,250,45,179,111,83,74,189,191,127,230,184,133,54,169,122,191,237,126,94,154,208,238,136,168,105,95,151,127,18,87,185,1,0,4,27,1,8,0,128,4,192,164,160,3,0,4,198,20,40,133,16,80,169,8,0,134,194,2,141,64,0,152,58,32,80,18,8,0,10,32,64,32,4,132,1,163,1,1,0,36,2,134,153,0,144,194,32,35,168,44,80,0,0,68,62,13,16,66,18,240,10,8,63,84,13,0,184,68,168,4,152,0,8,72,133,168,0,8,212,72,42,80,0,1,1,132,226,64,22,1,148,88,98,2,177,200,66,100,36,2,37,0,2,25,47,81,8,2,8,24,34,197,104,0,32,129,49,17,8,161,90,0,64,67,212,0,82,0,84,33,24,17,36,5,144,33,146,64,157,128,143,97,0,0,4,156,19,16,17,140,220,5,18,194,9,41,1,32,32,96,14,133,4,3,16,0,72,2,1,30,136,91,180,6,128,3,36,140,4,40,34,70,128,100,8,1,2,80,98,2,20,4,0,8,18,1,8,20,20,0,84,145,0,2,28,67,216,17,192,66,1,200,146,2,0,242,157,208,18,65,12,176,96,128,10,0,16,16,161,32,4,64,164,17,74,24,16,0,132,146,88,1,200,2,50,16,64,134,37,128,131,83,116,87,132,1,201,195,128,131,230,25,54,132,101,240,16,12,128,160,153,111,197,50,161,98,228,113,170,184,177,84,154,89,223,156,19,206,2,235,80,152,146,110,189,162,140,39,206,210,39,23,136,0,0,0,0,0,0,0,0,133,4,248,173,228,0,160,189,212,78,206,255,69,234,23,107,124,18,75,59,214,25,229,76,147,123,25,244,233,139,202,176,177,53,121,144,69,230,108,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0];
+global encoded_length: Field = 591;
```

Comment:
I know I already missed it in the previous PR, but why do we have two dencun blocks? Isn't it enough to have one? Also - even if we need to - let's keen naming convention consistent. 
some_blobGasUsed uses both snake and camel case

Reply:
Also - why did those values update in this PR?

Reply:
1: We have two Dencun blocks to test that when blobGasUsed is 0 we correctly rlp-encode it (that we unpad the value)
2: I forgot to regenerate fixtures after "5daacc5 : Add new fields to header for future Dencun upgrade" PR, so I've done it in this PR